migrations/V190__Add_System_Intake_Software_Acquisition_Table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TYPE software_acquisition_method AS ENUM ('CONTRACTOR_FURNISHED', 'FED_FURNISHED', 'ELA_OR_INTERNAL', 'NOT_YET_DETERMINED', 'OTHER');
+
+CREATE TABLE system_intake_software_acquisition (
+    id uuid PRIMARY KEY NOT NULL,
+    system_intake_id uuid NOT NULL REFERENCES system_intakes(id),
+    using_software TEXT,
+    acquisition_methods software_acquisition_method[],
```

Comment:
I might be oversimplifying, but it seems like there's only two real columns here that aren't metadata about the actual row, could this just be added to the intake table and save performing a join? I believe there was a mention that this was also a one-to-one relationship, right?

Reply:
It is one to one, the main reason i split it out into another table is that the we are likely to add more information (list of software products and other info) at some point.

I am not opposed to refactoring to combine into the intake if people think the current implementation is overkill.

Reply:
Thinking about this more and talking to @ClayBenson94, I am going to follow your rec and refactor this to have it all int eh system intake.

Thanks @mynar7

Reply:
Following back here, I just pushed this refactor to remote. Should be ready for review. Thank you both!